dashboard/app/controllers/api/v1/regional_partners_controller.rb

```diff
@@ -18,7 +18,7 @@ def capacity
 
   # GET /api/v1/regional_partners/find
   def find
-    zip_code = params[:zip_code]
+    zip_code = params[:zip_code] if RegexpUtils.us_zip_code?(params[:zip_code])
```

Comment:
Does this need to be initialized to `nil` otherwise, in order to be used on line 25?

Reply:
I wondered the same thing, and it worked okay, but I agree that it seems proper to initialise to `nil`, so I'll do that.